### PR TITLE
docs: update einstein faq

### PR DESCRIPTION
**Pull request by sbudhirajadoc**
remove a line of code being used to improve E4D. Per this -- https://trailhead.salesforce.com/trailblazer-community/feed/0D54V00007ZBmP7

### What issues does this PR fix or reference?
#<Insert GitHub Issue>, @W-15678451@

### Functionality Before
FAQ not updated.

### Functionality After
FAQ Updated.

### CODE REVIEW

File: docs/_articles/en/einstein/einstein-faq.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ See [Trusted AI from Salesforce](https://www.salesforceairesearch.com/trusted-ai
 
 **Will my code ever be shared outside of my development environment?**
 
-No. Salesforce treats your code as confidential information under your Main Service Agreement (MSA) and doesn't disclose it to other Salesforce customers or anyone outside of Salesforce. Some of your code and entity schema can be used to improve Einstein for Developers and train CodeGen. Due to the nature of machine learning, Einstein for Developers can generate output that resembles code that was used to train the model.
+No. Salesforce treats your code as confidential information under your Main Service Agreement (MSA) and doesn't disclose it to other Salesforce customers or anyone outside of Salesforce. Due to the nature of machine learning, Einstein for Developers can generate output that resembles code that was used to train the model.
```
Comment (kfidelak94) [inline docs/_articles/en/einstein/einstein-faq.md:20]: No.  Salesforce treats your code as confidential information under your Main Service Agreement (MSA) and doesn't disclose it to other Salesforce customers or anyone outside of Salesforce. Additionally, your code is not used to improve the product or train the global LLM.  For complete terms of use see: [https://developer.salesforce.com/tools/vscode/en/einstein/einstein-termsofuse](https://developer.salesforce.com/tools/vscode/en/einstein/einstein-termsofuse). Due to the nature of machine learning, Einstein for Developers can generate output that resembles code that was used to train the model.

File: docs/_articles/en/einstein/einstein-faq.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ See [Trusted AI from Salesforce](https://www.salesforceairesearch.com/trusted-ai
 
 **Will my code ever be shared outside of my development environment?**
```
Comment (kfidelak94) [inline docs/_articles/en/einstein/einstein-faq.md:18]: I think this question should be stated as:
"Will my code ever be shared outside of Salesforce?"

Comment (sbudhirajadoc) [reply docs/_articles/en/einstein/einstein-faq.md:18]: agreed. Updating.

**Review by kfidelak94 — APPROVED** [review]
This looks good.  Just a note that if/when we provide option for using customer code for training the model, this should be updated again.

File: docs/_articles/en/einstein/einstein-faq.md [hunk 3]
```diff
@@ -15,9 +15,11 @@ Einstein for Developers uses our trusted generative AI, CodeGen, to assist you t
 
 See [Trusted AI from Salesforce](https://www.salesforceairesearch.com/trusted-ai).
 
-**Will my code ever be shared outside of my development environment?**
+**Will my code ever be shared outside of Salesforce?**
 
-No. Salesforce treats your code as confidential information under your Main Service Agreement (MSA) and doesn't disclose it to other Salesforce customers or anyone outside of Salesforce. Some of your code and entity schema can be used to improve Einstein for Developers and train CodeGen. Due to the nature of machine learning, Einstein for Developers can generate output that resembles code that was used to train the model.
+No. Salesforce treats your code as confidential information under your Main Service Agreement (MSA) and doesn't disclose it to other Salesforce customers or anyone outside of Salesforce. Additionally, your code is not used to improve the product or train the global LLM. See [Additional Terms of Use for the Einstein for Developers Extensions (Beta)]( https://developer.salesforce.com/tools/vscode/en/einstein/einstein-termsofuse) for details.
```
Comment (jshackell-sfdc) [inline docs/_articles/en/einstein/einstein-faq.md:20]:
```suggestion
No. Salesforce treats your code as confidential information under your Main Service Agreement (MSA) and doesn't disclose it to other Salesforce customers or anyone outside of Salesforce. Additionally, Salesforce doesn't use your code to improve the product or train the global LLM. See [Additional Terms of Use for the Einstein for Developers Extensions (Beta)]( https://developer.salesforce.com/tools/vscode/en/einstein/einstein-termsofuse) for details.
```

**Review by jshackell-sfdc — APPROVED** [review]
See my one small edit.

**Pull request merged**